packages/lib/config/networks/hyperevm.ts

```diff
@@ -62,12 +62,12 @@ const networkConfig: NetworkConfig = {
   }),
   layerZeroChainId: 367,
   supportsVeBalSync: false,
-  // lbps: {
-  //   collateralTokens: [
-  //     '0x5555555555555555555555555555555555555555', // wHYPE
-  //     '0x69f8AFbC9DE9fD819eC78eDc553488a6B4269938', // USDC
-  //   ],
-  // },
+  lbps: {
+    collateralTokens: [
+      '0x5555555555555555555555555555555555555555', // wHYPE
+      '0xB8CE59FC3717ada4C02eaDF9682A9e934F625ebb', // USDT
```

Comment:
`USDC` on hyperEVM doesnt seem to exist in our token list from API. should we add it?

Reply:
I just added the token from the list in their page that looked familiar, but someone should review which ones we want as collateral on every network. Yesterday Xeonus posted this though, maybe those were different USDC:

<img width="462" height="194" alt="Screenshot 2025-07-22 at 11 00 44" src="https://github.com/user-attachments/assets/ec8e04de-c080-4372-8e03-50a9cb8b13b4" />